src/js/contexts/MessageContext/MessageContext.js

```diff
@@ -32,11 +32,26 @@ export const format = (options, messages) => {
       messageObj = messageObj[idPart];
     }
   });
+
   const message =
     (options.messages ? options.messages[baseId] : undefined) ||
     messageObj ||
     options.defaultMessage;
-  return message;
+
+  const values = options.values ? options.values : undefined;
```

Comment:
This doesn't really do anything except maybe convert null, 0 or '' to undefined.
You could just do
```
const { values } = options;
Also message could technically be undefined at this point (which is why I try to use the newer ? operator in the suggestion below.)